server/controllers/conversation.js

```diff
@@ -0,0 +1,48 @@
+const asyncHandler = require("express-async-handler");
+const Conversation = require("../models/Conversation");
+const Profile = require("../models/Profile");
+
+//For scenario where ownerId will contact other user Via profile
```

Comment:
It would be cleaner to remove this comment - 👌🏻

Reply:
comment removed